apps/desktop/src/components/v3/StackTab.svelte

```diff
@@ -93,6 +90,8 @@
 		border-right: 1px solid var(--clr-border-2);
 		overflow: hidden;
 		min-width: 100px;
+		flex: 0 0 auto;
```

Comment:
Here you are saying that tabs should not resize, which is not what we want. Can you drop this line?

Reply:
Yeah so this was actually necessary, but looking at it again, only the second part - the shrink: 0 shorthand, was the important part :+1: 

If you look at the screenshots in the desc above it still grew/grows.